### ISSUE TITLE
One ring to rule them all?

### ISSUE THREAD

**Issue by monocongo** [Ouranosinc/xclim#1273]
### Setup Information

- Xclim version:
- Python version:
- Operating System:

### Context

I am the maintainer of the [climate_indices](https://github.com/monocongo/climate_indices) package. I keep that project alive solely as a "labor of love" in order to support open science. I'm running out of love, but the number of users keeps increasing (now at 244 stars).

I recently looked at the xclim code again and I was thoroughly impressed -- you all have made enormous progress and xclim is clearly the superior package now. To maintain a separate package is a wasteful duplication of effort, and undermines one of our original goals which was to provide a measure of standardization by providing a set of "reference" implementations so as to eliminate so much reinvention of the wheel as it were.

Would there be any interest/benefit on your side for this kind of transition, i.e. folding climate_indices into xclim and documenting a transition path for users? If so I'd like to get that ball rolling with your guidance.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Zeitsperre** [comment]
Hi @monocongo!

Just based on initial reactions from the dev team, this is certainly something we would be interested in exploring. We would want to gauge the level of effort/support/changes that would be required to ensure that the features/indices of `climate_indices` are translated well into `xclim`.

Would you be open to having a conversation with us over Zoom or Microsoft Teams?

**Comment by huard** [comment]
Hi James, 

To build on Trevor's answer, I see two pathways for this: 
1. Keep `climate_indices` alive, meaning indices would be ported to xclim, then imported back into climate_indices so your user base sees little to no change to their workflows. 
2. Close down `climate_indices` while documenting how to transition to using xclim instead. 

The ICCLIM package followed option 1, and I believe it's an arrangement that works well.

**Comment by monocongo** [comment]
@Zeitsperre Yes, Trevor, I'm happy to work this out over one or more video conference calls. Both Teams and Zoom are fine for me.

@huard I think option 1 is best here as well, David, where climate_indices becomes a wrapper for xclim.

**Comment by pagecp** [comment]
I (and also @bzah if he is interested and available) could also participate in the call if you see some interests in having our experience feedbacks too.
